### PR TITLE
Added Password Visibility Toggle

### PR DESCRIPTION
**Pull request by kpchungdev**
Added password visibility toggle in suggested(better) way.

### CODE REVIEW

File: app/src/main/res/layout/connection_wizard_framabag_config_fragment.xml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:orientation="vertical"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
+              xmlns:tools="http://schemas.android.com/tools"
```
Comment (Strubbl) [inline app/src/main/res/layout/connection_wizard_framabag_config_fragment.xml]: Can you please keep the whitespaces as they were and only add your code line?

File: app/src/main/res/layout/connection_wizard_generic_config_fragment.xml [hunk 2]
```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:orientation="vertical"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
+              xmlns:tools="http://schemas.android.com/tools"
```
Comment (Strubbl) [inline app/src/main/res/layout/connection_wizard_generic_config_fragment.xml]: please fix whitespaces here, too

File: app/src/main/res/layout/connection_wizard_wallabagit_config_fragment.xml [hunk 3]
```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:orientation="vertical"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
+              xmlns:tools="http://schemas.android.com/tools"
```
Comment (Strubbl) [inline app/src/main/res/layout/connection_wizard_wallabagit_config_fragment.xml]: and here too

**Comment by Strubbl** [comment]
Travis is complaining now: "Could not list contents of '/home/travis/build/wallabag/android-app/app/src/main/res/values-en/strings.xml'. Couldn't follow symbolic link."
https://travis-ci.org/wallabag/android-app/builds/409613739#L2241

**Comment by kpchungdev** [comment]
Originally, I couldn't run Wallabag without putting those strings in en-strings.xml so I manually put them in and accidentally committed that file.

**Comment by Strubbl** [comment]
@kpchungdev Would you mind to re-open a new PR with your changes, but without the symlink changed?

**Pull request closed without merging**